options/locale/locale_en-US.ini

```diff
@@ -691,7 +691,7 @@ emails = Email Addresses
 manage_emails = Manage Email Addresses
 manage_themes = Select default theme
 manage_openid = Manage OpenID Addresses
-email_desc = Your primary email address will be used for notifications and other operations.
+email_desc = Your primary email address will be used for notifications and Git operations made using the web interface.
```

Comment:
The sentence reads strange, because IMO:

1. Not only "Git operations made using the web interface", but all related "Git operations"
2. Not only notifications, but I think there are also emails for verification or password recovery?

Reply:
> Not only "Git operations made using the web interface", but all related "Git operations"

If I make a commit using a secondary email that is not primary, my primary email address will not be used.

> Not only notifications, but I think there are also emails for verification or password recovery?

Good point.

Reply:
> > Not only "Git operations made using the web interface", but all related "Git operations"
> 
> If I make a commit using a secondary email that is not primary, my primary email address will not be used.

I see your point and I misunderstood it before. I think it means "this email will be used as the committer email for web Git operations"

Reply:
I'll change it to web-based, I think it's very much clear as day and I don't think it will confuse more advanced users that would presumably use the `tea` CLI tool later on.

Reply:
32ce04ce28078464d5248162e3ab479d1ba23394 makes the text much longer, but I think that the consequences of email-related settings are very well defined. I hope that we won't have to revisit this later down the line.

Reply:
The sentence `will be used for .... and, provided that it is not hidden, web-based Git operations` reads a little difficult to me. But I am not a native English speaker, so I can't comment on it (aka, no more option from my side).

Reply:
I don't think I can come up with a better combination of being accurate, easy to read and being transparent.